flink-runtime/src/test/java/org/apache/flink/runtime/zookeeper/ZooKeeperStateHandleStoreTest.java

```diff
@@ -79,24 +68,24 @@
  *   <li>Correct ordering of ZooKeeper and state handle operations
  * </ul>
  */
-public class ZooKeeperStateHandleStoreTest extends TestLogger {
+class ZooKeeperStateHandleStoreTest extends TestLogger {
 
     private static final ZooKeeperTestEnvironment ZOOKEEPER = new ZooKeeperTestEnvironment(1);
```

Comment:
`ZooKeeperExtension` is the JUnit5 way to do things with a ZK test environment. That would also fix FLINK-26341.

Reply:
Updated to `ZooKeeperExtension`.